packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.js

```diff
@@ -88,13 +88,11 @@ class BpkScrollableCalendarGridList extends React.Component {
     );
   }
 
-  renderList({ width, height }) {
```

Comment:
What's happening here?

Reply:
AutoSizer calls its child with a single object `{ width, height }`.  We were accepting `(width, height)` without destructuring it which means:
```js
width = { width: xxx, height: xxx };
height = undefined;
```
So that wasn't really correct. Now it's passing width and height properly to renderList. It's a local method so there's not much reason to wrap it in an object, if we need to change the interface we can do it right in that file.

Reply:
So it was previously broken yeah? 😱 

Whats the before & after behaviour?

Reply:
It was broken, yeah, but I didn't see any change in behaviour. It might change something about how it decides whether to render an item in the grid or not.

Reply:
Ok doke 😅